zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/handlers/ListViewProcessInstanceFromProcessInstanceHandler.java

```diff
@@ -152,6 +152,9 @@ public void flush(
     if (entity.getEndDate() != null) {
       updateFields.put(ListViewTemplate.END_DATE, entity.getEndDate());
     }
+    if (entity.getProcessVersionTag() != null) {
+      updateFields.put(ListViewTemplate.PROCESS_VERSION_TAG, entity.getProcessVersionTag());
```

Comment:
🔧 should we increase unit test coverage to cover this field too?

Reply:
Thanks, you're right! I extended the tests.